internal/app/config.go

```diff
@@ -44,10 +44,10 @@ type Configuration struct {
 	// ConditionDefinitions holds one or more condition definitions the conditionorc API, orchestrator support.
 	ConditionDefinitions rctypes.Definitions `mapstructure:"conditions"`
 
-	// ServerserviceOptions defines the serverservice client configuration parameters
+	// FleetDBAPIOptions defines the serverservice client configuration parameters
 	//
 	// This parameter is required when StoreKind is set to serverservice.
-	ServerserviceOptions ServerserviceOptions `mapstructure:"serverservice"`
+	FleetDBAPIOptions FleetDBAPIOptions `mapstructure:"serverservice"`
```

Comment:
change the `mapstructure` tag? Or capture a task to do so later?

Reply:
Yep, I intend to take this on separately - since it involves changing the k8s templates, and that can be done in a separate sweep across the apps.